### DISCUSSION TITLE
code-server redirects to /:code/:code/:code/:code/

### DISCUSSION THREAD

**Discussion by saloricardo** [coder/code-server#6263] [Q&A]
Hello everyone,

For some reason my code-server started redirecting to /:code/:code/:code/:code/:code/:code/:code/:code/:code/

I've already tried uninstalling the extensions, but it keeps doing that, I always need to access it by pointing to the workspace I want? workspace=path
 Otherwise, if I try to access http://localhost:8888/ directly, it adds those paths to the address.

Has anyone gone through this? Do you know how to fix?

Thanks!

**Comment by code-asher** [comment]
Odd, it should only redirect to the last opened workspace or 
directory.  What do you have in 
~/.local/share/code-server/coder.json?  That is where the last 
workspace/directory is stored.

Do you have anything in front of code-server like a reverse proxy 
that could be doing this?

**Comment by saloricardo** [comment]
I looked in this file that you mentioned and everything seems to be right, the path is correct.

But I solved the problem by creating another Workspace, and with a different name, if it was the same name, the problem continued. Super weird, but OK.

**Reply by code-asher** [reply]
Very strange indeed!  Maybe the problem could be client-side; perhaps a redirect was cached by your browser.
